parachains/runtimes/assets/statemine/src/weights/pallet_assets.rs

```diff
@@ -66,115 +66,113 @@ impl<T: frame_system::Config> pallet_assets::WeightInfo for WeightInfo<T> {
 	/// The range of component `c` is `[0, 5000]`.
 	/// The range of component `s` is `[0, 5000]`.
 	/// The range of component `a` is `[0, 500]`.
-	fn destroy(c: u32, s: u32, a: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 103_000
-			.saturating_add(Weight::from_ref_time(17_019_000 as u64).saturating_mul(c as u64))
-			// Standard Error: 103_000
-			.saturating_add(Weight::from_ref_time(18_891_000 as u64).saturating_mul(s as u64))
-			// Standard Error: 1_038_000
-			.saturating_add(Weight::from_ref_time(32_265_000 as u64).saturating_mul(a as u64))
-			.saturating_add(T::DbWeight::get().reads(5 as u64))
-			.saturating_add(T::DbWeight::get().reads((2 as u64).saturating_mul(c as u64)))
-			.saturating_add(T::DbWeight::get().reads((2 as u64).saturating_mul(s as u64)))
-			.saturating_add(T::DbWeight::get().reads((1 as u64).saturating_mul(a as u64)))
-			.saturating_add(T::DbWeight::get().writes(2 as u64))
-			.saturating_add(T::DbWeight::get().writes((2 as u64).saturating_mul(c as u64)))
-			.saturating_add(T::DbWeight::get().writes((2 as u64).saturating_mul(s as u64)))
-			.saturating_add(T::DbWeight::get().writes((1 as u64).saturating_mul(a as u64)))
+	fn destroy(c: u32, s: u32, _a: u32, ) -> Weight {
+		Weight::from_ref_time(71_841_919_000 as u64)
+			// Standard Error: 138_450
+			.saturating_add(Weight::from_ref_time(10_684_701 as u64).saturating_mul(c as u64))
+			// Standard Error: 138_450
+			.saturating_add(Weight::from_ref_time(11_056_248 as u64).saturating_mul(s as u64))
+			.saturating_add(T::DbWeight::get().reads(10505 as u64))
```

Comment:
odd change?

Reply:
@EgorPopelyaev 
The numbers for destroy are wrong, and I do not see any changes for destroy itself and its benches between 9300 and prev version.
Probably we need to rerun.

@ggwpez have you seen anything like this?

Reply:
Yea that fixes it. Not sure if you want to cherry-pick or how its handled.

Reply:
Okay it is merged in Substrate.